docs/advanced-entity-analytics/analyze-risk-score-data.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[analyze-risk-score-data]]
+== View and analyze risk score data
+
+The {security-app} provides several options to monitor the change in the risk posture of hosts and users from your environment. We recommend that you prioritize alert triaging to identify anomalies or abnormal behavior patterns.
```

Comment:
> We recommend that you prioritize alert triaging to identify anomalies or abnormal behavior patterns.

This feels out of place here. Maybe highlight `Alert prioritization using entity risk` under the Alert section instead?

Reply:
We could put this in a `TIP:` before we list after we list the options (the jump list Joe recommended below). 

> TIP: We recommend that you prioritize alert triaging to identify anomalies or abnormal behavior patterns.
 
I'd include an `xref` link to the alert triage section within the tip. 

@SourinPaul - I'd probably label this section `Alert triaging` to keep it simple.